downstream/modules/automation-hub/proc-create-synclist.adoc

```diff
@@ -12,11 +12,22 @@ By default, all Red Hat Certified collections are included in your initial organ
 
 * You have a valid Ansible Automation Platform subscription.
 * You have Organization Administrator permissions for cloud.redhat.com.
+* Ensure that the following domain names are part of either the firewall or the proxy's allowlist for successful connection and download of collections from {HubName} or Galaxy server:
```

Comment:
{HubName} and {HubNameStart} aren't rendering properly in my local docs build. I think we need to create a PR to main to fix this.

Try adding the following line to downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```
include::attributes/attributes.adoc[]
```

Reply:
In main?

Reply:
Yes. I've just created a PR: #444

Reply:
Once that's merged, you can rebase the backport PR branch against upstream/main and then update this PR.